docs/extensibility/custom-hosting-integration.md

```diff
@@ -487,5 +487,8 @@ In the custom resource tutorial, you learned how to create a custom .NET Aspire
 
 ## Next steps
 
+> [!div class="nextstepaction"]
+> [Create non-container custom resources](custom-non-container-resource.md)
```

Comment:
I wouldn't consider this really a next step, it's more like an alternative approach. I'd consider removing this link here.